packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -213,52 +215,32 @@ export abstract class Chart extends Observable implements AgChartInstance {
         this.scene.download(fileName, fileFormat);
     }
 
+    @Validate(OBJECT)
     padding = new Padding(20);
 
-    _seriesArea = new SeriesArea();
-    get seriesArea() {
-        return this._seriesArea;
-    }
-    set seriesArea(newArea: SeriesArea) {
-        if (!newArea) {
-            this._seriesArea = new SeriesArea();
-        } else {
-            this._seriesArea = newArea;
-        }
-    }
+    @Default(() => new SeriesArea())
```

Comment:
Nit: Do we need this and the inline assignment on the next line?

Reply:
You may be right, I'll test it.

Reply:
it seems to be need because `Default` only triggers on `set`.